### ISSUE TITLE
emoji/emoticon name when hovering

### ISSUE THREAD

**Issue by lanodan** [tootsuite/mastodon#886]
Currently: When you hover the mouse hover a emoji/emoticon in a toot nothing appears
Wanted: When you hover the mouse hover a emoji/emoticon in a toot it’s name appears

Edit: For toots, not just emoji pop-in.

**Comment by john-light** [comment]
appears to work for me 

![emojiname](https://cloud.githubusercontent.com/assets/9424721/24690581/9cf6bab4-1982-11e7-8249-a16dc205e349.png)

**Comment by braak** [comment]
The name only appears in the emoji selector but not in toots. Adding the short name as alt text might also help with accessibility.

**Comment by clochix** [comment]
This is also an accessibility issue. Every image should have an alternative text.

We may use something like `alt = emojione.toShort(unicode)` in https://github.com/tootsuite/mastodon/blob/master/app/assets/javascripts/components/emoji.jsx#L16

**Comment by unarist** [comment]
This has been fixed at #2784, and released as v1.4.1.
